components/sections/ContactUs.jsx

```diff
@@ -1,11 +1,11 @@
 import LargeButton from "../ui/buttons/LargeButton";
-import { Heading3 } from "../ui/Typography";
+import { Heading1 } from "../ui/Typography";
 
 const ContactUs = () => {
     return (
         <section className="bg-primary-800 mx-0.5 rounded-8 gap-16 xl:px-0 px-5 py-5.5 md:py-16 flex flex-col items-center">
             <div className="xl:w-3.73/5">
-                <Heading3 className="text-center text-white text-11.75">Ready to talk about your project?</Heading3>
+                <Heading1 className="text-center text-white text-11.75">Ready to talk about your project?</Heading1>
```

Comment:
Here you used the `Heading1` component which has a `font-size: 59px` while we should be having a `font-size:47px` basically you could use the `Heading2` instead of `Heading1`